packages/build/src/bundle.ts

```diff
@@ -45,41 +45,22 @@ export class BuildBundler extends AsyncTransformStream<File, File> {
 
     this._buildAnalyzer = buildAnalyzer;
 
-    const {
-      analyzer,
-      excludes,
-      inlineCss,
-      inlineScripts,
-      rewriteUrlsInTemplates,
-      sourcemaps,
-      stripComments,
-      urlMapper
-    } = options;
-    let {strategy} = options;
-
-    const urlLoader =
-        new FileMapUrlLoader(this.files, analyzer || buildAnalyzer.analyzer);
-    const forkedAnalyzer =
-        (analyzer || buildAnalyzer.analyzer)._fork({urlLoader});
-
-    if (strategy === undefined && this.config.shell !== undefined) {
-      strategy =
-          generateShellMergeStrategy(forkedAnalyzer.resolveUrl(urlFromPath(
+    const bundlerOptions = {...options};
```

Comment:
I'll try not to commit any code this time, but clarification question, what's the benefit of having a copy of the `options` object here rather than just working with the one supplied?

Reply:
I don't want the function to mutate the object handed to it.  I swap the analyzer for a forked version.